### PR TITLE
fixing kill processes on host

### PR DESCRIPTION
**Pull request by kensipe**
fixing kill processes to align with run_command changes and to actually make it work

### CODE REVIEW

**Comment by kensipe** [comment]
I left the original print code, however it seems verbose.  if you have thoughts, let me know.

**Comment by sschneid** [comment]
Added a slight tweak to the output.  If I log into the host and start a `tee -a` as the `core` user, then `kill_process_on_host(<host>, 'tee -a')`:

```
┍┈┈ ✓: test_kill_process_on_host.py::test_kill_process_on_host
┊ 》35.161.79.164 $ ps aux | grep -v grep | grep 'tee -a'
┊
┊ core     20338  0.0  0.0   4276   716 pts/0    S+   21:55   0:00 tee -a
┊
┊ 》35.161.79.164 $ sudo kill -9 20338
┊
┊ Killed pid: 20338
```

**Comment by colin-msphere** [comment]
retest this please

File: shakedown/dcos/agent.py [hunk 1]
```diff
@@ -83,31 +83,16 @@ def kill_process_on_host(
...
-        ip, "proc_id=$(ps ax | grep '" + pattern + "' | awk '{{ print $1 }}') && echo $proc_id"
-    )
-    pids = str(result).split()
-
-    print("\n{}pattern {} matched following PIDs: {}\n".format(
-        shakedown.cli.helpers.fchr('>>'),
-        pattern,
-        pids
-    ))
+    status, stdout = run_command_on_agent(hostname, "ps aux | grep -v grep | grep '" + pattern + "'")
```
Comment (colin-msphere) [inline shakedown/dcos/agent.py]: Can this change to use the `"{}".format(...)` style as is used below?

**Comment by sschneid** [comment]
#yolo

**Pull request merged**